aws-rds-optiongroup/aws-rds-optiongroup.json

```diff
@@ -0,0 +1,204 @@
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "OptionName"
+      ]
+    },
+    "OptionSetting": {
+      "description": "The OptionSetting property type specifies the value for an option within an OptionSetting property.",
+      "type": "object",
+      "properties": {
+        "Name": {
+          "description": "The name of the option that has settings that you can set.",
+          "type": "string"
+        },
+        "Value": {
+          "description": "The current value of the option setting.",
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "Tag": {
+      "description": "A key-value pair to associate with a resource.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "Key": {
+          "type": "string",
+          "description": "The key name of the tag. You can specify a value that is 1 to 127 Unicode characters in length and cannot be prefixed with aws:. You can use any of the following characters: the set of Unicode letters, digits, whitespace, _, ., /, =, +, and -. ",
+          "minLength": 1,
+          "maxLength": 127
+        },
+        "Value": {
+          "type": "string",
+          "description": "The value for the tag. You can specify a value that is 1 to 255 Unicode characters in length and cannot be prefixed with aws:. You can use any of the following characters: the set of Unicode letters, digits, whitespace, _, ., /, =, +, and -. ",
+          "minLength": 1,
+          "maxLength": 255
+        }
+      },
+      "required": [
+        "Key",
+        "Value"
+      ]
+    }
+  },
+  "properties": {
+    "OptionGroupName": {
+      "description": "Specifies the name of the option group.",
+      "type": "string",
+      "pattern": "^[a-zA-Z]{1}(?:-?[a-zA-Z0-9]){0,254}$"
+    },
+    "OptionGroupDescription": {
+      "description": "Provides a description of the option group.",
+      "type": "string"
+    },
+    "EngineName": {
+      "description": "Indicates the name of the engine that this option group can be applied to.",
+      "type": "string",
+      "enum": [
+        "aurora",
+        "aurora-mysql",
+        "aurora-postgresql",
+        "mariadb",
+        "mysql",
+        "oracle-ee",
+        "oracle-se2",
+        "oracle-se1",
+        "oracle-se",
+        "postgres",
+        "postgresql-license",
+        "sqlserver-ee",
+        "sqlserver-se",
+        "sqlserver-ex",
+        "sqlserver-web"
+      ]
+    },
+    "MajorEngineVersion": {
+      "description": "Indicates the major engine version associated with this option group.",
+      "type": "string"
+    },
+    "OptionConfigurations": {
+      "description": "Indicates what options are available in the option group.",
+      "type": "array",
+      "uniqueItems": true,
+      "insertionOrder": false,
```

Comment:
this will default implementation to a set interface. is it backward compatible?

Reply:
Removed.